src/wallet/wallet.cpp

```diff
@@ -1193,19 +1193,19 @@ void CWallet::LoadToWallet(CWalletTx& wtxIn)
     }
 }
 
-bool CWallet::AddToWalletIfInvolvingMe(const CTransactionRef& ptx, CWalletTx::Status status, const uint256& block_hash, int posInBlock, bool fUpdate)
+bool CWallet::AddToWalletIfInvolvingMe(const CTransactionRef& ptx, CWalletTx::Confirmation confirm, bool fUpdate)
```

Comment:
In commit "Replace CWalletTx::SetConf by Confirmation initialization list" (d0c4ce686ba388b9fefff2b153358e33a042ee80)

Not a big deal, but probably better to pass with reference type `const CWalletTx::Confirmation&` to avoid copies when this is called. Similarly for other functions taking confirm arguments.